runtime/test/products-test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
```

Comment:
Update copyright date (and same rest of change for new files).

Reply:
done